NEWS.md

```diff
@@ -1,5 +1,26 @@
 # Release notes
 
+## Version 0.5.2 (2024-08-26)
+
+### Bugfix
+
+* Fixed a work around for a bug related to colors on plots not being updated upon selection (due to bug in Makie).
+* Fixed bug for plotting `ScenarioProfile`s.
+* Fixed bug when selecting periods from the menus for a non-tensorial time structure
+
+### Enhancement
+
+* The extraction of JuMP data is done during initialization of the GUI to improve performance during use.
+* Tests have been improved.
+
+### Feature
+
+* The total opex and capex costs may now be plotted over strategic periods where both quantities may be scaled to a year or the whole strategic period using the boolean input arguments `scale_tot_opex` and `scale_tot_capex`. The calculations are based on the variables listed in `src/descriptive_names.yml` under `total.opex_fields` and `total.capex_fields`. Partly solves [Issue #6](https://github.com/EnergyModelsX/EnergyModelsGUI.jl/issues/6).
+* Elements with added investments are now indicated by coloring the label red. What constitutes an added investment is positive values of the variables listed in `src/descriptive_names.yml` under `investment_indicators`. Partly solves [Issue #6](https://github.com/EnergyModelsX/EnergyModelsGUI.jl/issues/6).
```

Comment:
I see the red labels and the investment periods. The question is, whether we need to provide as well the investment periods within the main graph. Depending on the number of nodes, it can get quite confusing.

Reply:
In what sense would you say it is confusing? Maybe in the case of overlapping labels (due to long strings)?

Reply:
Yes. We only have smaller regional cases. If we have, let's say, 50 nodes in a region, then it only adds significantly to a confusion.

Reply:
ok, I will remove this part of the label.